### PR TITLE
Fix NPE mapping banner DAO if a banner has not been defined

### PR DESCRIPTION
**Pull request by johnaohara**

### CODE REVIEW

File: horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/BannerServiceImpl.java [hunk 1]
```diff
@@ -70,7 +70,8 @@ public void set(Banner dto) {
    @PermitAll
    @Override
    public Banner get() {
-     return BannerMapper.from(getBanner());
+      BannerDAO bannerDAO = getBanner();
```
Comment (jesperpedersen) [inline horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/BannerServiceImpl.java:73]: Extra space

File: horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/BannerServiceImpl.java [hunk 2]
```diff
@@ -70,7 +70,8 @@ public void set(Banner dto) {
...
+     return bannerDAO == null ? null : BannerMapper.from(getBanner());
```
Comment (jesperpedersen) [inline horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/BannerServiceImpl.java:74]: `getBanner()` -> `bannerDAO`

**Review by jesperpedersen — CHANGES_REQUESTED** [review]

File: horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/BannerServiceImpl.java (hunk 2 above)
Comment (johnaohara) [reply horreum-backend/src/main/java/io/hyperfoil/tools/horreum/svc/BannerServiceImpl.java:74]: thats what i get for rushing

**Comment by jesperpedersen** [comment]
Merged.

Thanks for your contribution !

**Pull request merged**